### PR TITLE
recommend nesting resources

### PR DESCRIPTION
**Pull request by benseligman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,13 +40,20 @@ Example:
...
+    render :json => { :errors => member.errors.messages }, :status => :unprocessable_entity
   end
 end
 ```
+* In general, nest resources in responses under a parent node. This makes the api extensible if we need to send additional entities in the response. For example:
+```ruby
+# bad
+render member.errors.full_messages # if we want to add more information other than errors to the response, we don't have a place to put it
 
+# bad
```
Comment (maulin) [inline README.md]: good?

Comment (cgadea) [inline README.md]: Should this be `#good` ?

**Comment by maulin** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (benseligman) [inline README.md]: EVERYTHING IS BAD

**Comment by cgadea** [comment]
lgtm

**Pull request closed without merging**